docs/buildkit/index.md

```diff
@@ -0,0 +1,154 @@
+names (`git`, `php`, `node`, `mysql`, `mysqldump`, etc). In some environments,
+you may need to enable these commands by configuring `PATH` -- this is especially
+true for MAMP, XAMPP, and other downloaded packages.
+(See, e.g., [Setup Command-Line PHP](http://wiki.civicrm.org/confluence/display/CRMDOC/Setup+Command-Line+PHP).)
+
+Once the pre-requisites are met, download buildkit to `~/buildkit`:
+
+```bash
+git clone https://github.com/civicrm/civicrm-buildkit.git ~/buildkit
+cd ~/buildkit
+./bin/civi-download-tools
+```
+
+### Troubleshooting
+
+* Nodejs version too old or npm update does not work
+
+Download the latest version from nodejs.org and follow to their instructions
+
+* Nodejs problems
+
+It might be handy to run
+
+```bash
+npm update
+npm install fs-extra
+```
+
+
+## Configuring buildkit after installation
+
+Buildkit includes many CLI commands in the `bin/` folder.
+
+You may execute the commands directly (e.g.  `./bin/civix` or `/path/to/buildkit/bin/civix`).  However, this would
+become very cumbersome.  Instead, you should configure the shell's `PATH` to recognize these commands automatically.
+
+!!! tip
+    Throughout this document, we will provide examples which assume that buildkit was downloaded to `/path/to/buildkit`. Be sure to adjust the examples to match your system.
+
+### Persistently add buildkit to `PATH` (typical)
+
+If you want to ensure that the buildkit CLI tools are always available, then:
+
+1. Determine the location of your shell configuration file. This is usually `~/.bashrc`, `~/.bash_profile`, or
+`~/.profile`.
+1. At the end of the file, add `export PATH="/path/to/buildkit/bin:$PATH"`
+1. Close and reopen the terminal.
+1. Enter the command `which civibuild`. This should display a full-path. If nothing appears, then retry the steps.
+
+### Temporarily switch shell configuration (advanced)
+
+Buildkit includes specific versions of some fairly popular tools (such as `drush`, `phpunit`, and `wp-cli`), and it's
+possible that you have already installed other versions of these tools.
+
+By design, buildkit can coexist with other tools, but you must manually manage the `PATH`.
+Whenever you wish to use buildkit, manually run a command like, e.g.:
+
+```bash
+export PATH=/path/to/buildkit/bin:$PATH
+```
+
+To restore your normal `PATH`, simply close the terminal and open a new one.
+
+Each time you open a new terminal while working on Civi development, you would need to re-run the `export` command.
+
+
+
+## Upgrading buildkit
+
+New versions of buildkit are likely to include new versions of tools. The
+new tools will download automatically when you first run "civibuild".
+If you prefer to download explicitly, then re-run "civi-download-tools".
+
+The configurations and tools in buildkit are periodically updated. To get the latest, simply run:
+
+```bash
+cd ~/buildkit
+git pull
+./bin/civi-download-tools
+```
+
+### v14.05.0 => v14.06.0
```

Comment:
Seems weird to maintain a changelog in a separate repo. Perhaps this could be a file `civicrm-buildkit.git/CHANGELOG.md`, and then link from the dev docs?